micro-core/src/main/java/com/aol/micro/server/module/ConfigurableModule.java

```diff
@@ -18,40 +18,57 @@
 import javax.servlet.ServletContextListener;
 import javax.servlet.ServletRequestListener;
 
-import com.aol.cyclops.data.collections.extensions.persistent.PMapX;
-import com.aol.cyclops.data.collections.extensions.persistent.PSetX;
-import com.aol.cyclops.data.collections.extensions.persistent.PStackX;
 import com.aol.micro.server.auto.discovery.CommonRestResource;
 import com.aol.micro.server.servers.model.ServerData;
 import com.aol.micro.server.utility.HashMapBuilder;
 
+import cyclops.collections.immutable.PMapX;
+import cyclops.collections.immutable.PSetX;
+import cyclops.collections.immutable.PStackX;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.experimental.Wither;
 
 
 @Builder
 @AllArgsConstructor
-@Wither
+
 public class ConfigurableModule implements Module{
 
+	@Wither
 	private final Set<Object> jaxRsResourceObjects;
+    @Wither
```

Comment:
It's strange, I'm not sure why. Looks fine in IntelliJ - should be spaces not tabs too. Will play around a bit more.